.gitignore

```diff
@@ -51,83 +51,23 @@ jabref.xml
 
 
 
-# Created by https://www.gitignore.io/api/gradle,java,jabref,intellij,eclipse,netbeans,windows,linux,macos
```

Comment:
More specifically, I added `snapcraft` to the API call. Instead of sorting the updates out, I just replaced the whole output

Created by https://www.gitignore.io/api/gradle,java,jabref,intellij,eclipse,netbeans,windows,linux,macos,snapcraft